### PR TITLE
Recipe app

### PR DESCRIPTION
**Pull request by Kalunge**
## Recipe app

- follow the layout of the wireframes provided and personalize the rest of the design including colours, typographies, spacings
- Login page and registration page:
    - Should be built with Devise.
- Food list:
    - Should display a list of food as in the wireframe.
    - Should lead to the form that allows users to add new food.
- Recipes list:
    - Should display a list of recipes created by the logged-in user as in the wireframe.
    - Should lead to recipe details.
    - If the user is the owner of the recipe, should allow the user to delete it.
- Public recipe list:
    - Should display a list of all public recipes ordered by newest as in the wireframe.
    - Should lead to recipe details.
- Recipe details:
    - If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
    - If the user is the owner of the recipe, should lead to the form that allows the user to add new food.
- General shopping list view (**do not implement if in a group of 3**):
    - Should show the list of food that is missing for all recipes of the logged-in user.
    - Should count the total food items and total price of the missing food.
- Make sure there are no N+1 queries happening.
- Create a navigation menu that allows users to open all of the pages you created.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+- Clone this repository by running `git clone https://github.com/DammyShittu/Recipe-App.git` in your command line.
+
+- Navigate to the repository by running `cd Recipe-App`.
+
+- Run `yarn install` to install all dependencies
+
+- Run `rails db:create`, and `rails db:migrate`  to create and migrate your database.
+
+- Run `rails s` and go to http:localhost:3000 to view the app.
```
Comment (Dipeshtwis) [inline README.md:25]: [OPTIONAL]

- I would recommend adding a command to test the application. [ `rspec`]. As this is a small change, I'm expecting that you will add this in your README, hence I'm approving this pull request.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @Kalunge & @DammyShittu, 

### Status: Approved ✔️

:+1:  Professional repo, with good commit message and PR title, keep up the good work :muscle:
:+1: The app is working well. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**



**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kalunge** [comment]
Thanks, will definitely add that

**Comment by DammyShittu** [comment]
Thank you for the review @Dipeshtwis

**Pull request merged**